widgets/qt_view_laser_mode.py

```diff
@@ -0,0 +1,54 @@
+import sys
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
+import qt_custom_decorations
+
+
+class InitializeComboButton(QWidget):
+    def __init__(self, parent, button_name, add_line_break=False, can_disable=False):
+        super(QWidget, self).__init__(parent)
+
+        self.parent = parent
+        self.button_name = button_name
+        self.add_line_break = add_line_break
+        self.can_disable = can_disable
+
+        if self.can_disable:
+            self.button_state = False
```

Comment:
else? `self.button_state` wouldn't be defined? is that intentional?

Reply:
yes, this prevents the view button from being able to disable the widgets.

Reply:
ha, i think we can refactor this into something simpler